files/build_templates/init_cfg.json.j2

```diff
@@ -38,7 +38,7 @@
 {%- set features = [("bgp", "{% if not DEVICE_RUNTIME_METADATA['ETHERNET_PORTS_PRESENT'] or ('CHASSIS_METADATA' in DEVICE_RUNTIME_METADATA and DEVICE_RUNTIME_METADATA['CHASSIS_METADATA']['module_type'] in ['supervisor']) %}disabled{% else %}enabled{% endif %}", false, "enabled"),
                    ("database", "always_enabled", false, "always_enabled"),
                    ("lldp", "enabled", true, "enabled"),
-                   ("pmon", "enabled", true, "enabled"),
+                   ("pmon", "enabled", "{% if 'type' in DEVICE_METADATA['localhost'] and DEVICE_METADATA['localhost']['type'] == 'SpineRouter' %}False{% else %}True{% endif %}", "enabled"),
```

Comment:
Why use titlecase instead of lowercase here?

Reply:
This is to align with https://github.com/sonic-net/sonic-host-services/pull/135
In this change https://github.com/sonic-net/sonic-host-services/blob/ca6b3cdb9a56604765ada4ac67234c117f0fb5ad/scripts/featured#L79 , the host-service takes titlecase only and evaluate them with python eval.